stream/vibe/stream/wrapper.d

```diff
@@ -242,9 +242,13 @@ class ConnectionProxyStream : ConnectionStream {
 	after a call to front. This property allows the range to be used in
 	request-response scenarios.
 */
-struct StreamInputRange {
+deprecated("Use streamInputRange() instead.")
+StreamInputRange!OutputStream StreamInputRange()(InputStream stream) { return StreamInputRange!InputStream(stream); }
```

Comment:
Looking at the diff, it looks like this function didn't exists before, but is added as deprecated ? What am I missing ?

Reply:
This is a backwards-compatibility replacement of the old constructor, which was simply used as `StreamInputRange(stream)`. Since the struct is now wrapped in a template, that syntax is otherwise not possible anymore.